Fabulous.XamarinForms/src/Fabulous.XamarinForms.Core/ViewUpdaters.fs

```diff
@@ -675,6 +675,13 @@ module ViewUpdaters =
             let realTarget = (target :> Xamarin.Forms.IShellContentController).Page
             if realTarget <> null then currValue.UpdateIncremental(prevValue, realTarget)            
         | ValueSome _, ValueNone -> target.ContentTemplate <- null
+
+    let updateShellNavBarHasShadow prevValueOpt currValueOpt target =
+        match prevValueOpt, currValueOpt with
+        | ValueSome prevValue, ValueSome currValue when prevValue = currValue -> ()
+        | ValueNone, ValueNone -> ()
+        | _, ValueSome currValue -> Shell.SetNavBarHasShadow(target, currValue)
+        | ValueSome _, ValueNone -> Shell.SetNavBarHasShadow(target, true)
```

Comment:
The default value of `NavBarHasShadow` is dependent of the platform. If it's Android, it is `true`, `false` otherwise.